### PR TITLE
feat(template): Add templates for different chaos experiments

### PR DESCRIPTION
**Pull request by uditgaurav**
Signed-off-by: Udit Gaurav <uditgaurav@gmail.com>

- Add templates for different chaos experiments. These templates are useful in several ways:
  - Anyone can remotely use these templates to run chaos experiment
  - It will be helpful to test the changes in a simpler way
  - Can be integrated with one's CI

 How to use it?
The templates can be placed in the extends of `gitlab-ci.yml`

### CODE REVIEW

File: templates/container-kill-template.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+---
+variables:
+  APP_NS: default
+  APP_LABEL: "run=nginx"
```
Comment (ksatchit) [inline templates/container-kill-template.yml]: can we use placeholders as default vars values over nginx labels

File: templates/container-kill-template.yml [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  ##only deployment is supported
+  APP_KIND: deployment 
+  TARGET_CONTAINER: "nginx"
```
Comment (ksatchit) [inline templates/container-kill-template.yml]: placeholder?

File: templates/container-kill-template.yml [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+  EXPERIMENT_IMAGE: litmuschaos/ansible-runner:latest
+  OPERATOR_IMAGE: litmuschaos/chaos-operator:latest
+  RUNNER_IMAGE: litmuschaos/chaos-runner:latest
+
+.container_kill_template:
+  image: uditgaurav/gitlab-template:latest
```
Comment (ksatchit) [inline templates/container-kill-template.yml]: we need a mayadata-io image for this. Dockerfile too can be placed (I suppose in ci-lib repo)

File: templates/node-cpu-hog-template.yml [hunk 4]
```diff
@@ -0,0 +1,17 @@
+---
+variables:
+  APP_NS: default
+  APP_LABEL: "run=nginx"
+  ##only deployment is supported
+  APP_KIND: deployment 
+  TOTAL_CHAOS_DURATION: "60"
+  NODE_CPU_CORE: "2"
+  EXPERIMENT_IMAGE: litmuschaos/ansible-runner:latest
```
Comment (ksatchit) [inline templates/node-cpu-hog-template.yml:9]: wondering if we should expose only experiment image (while keeping operator and runner internal. By that I mean, keep the option for override open, don''t expose them in templates). Operator and runner are infra components of litmus - while experiment can be a dev thing. Infra can be abstracted away by default.

Comment (uditgaurav) [reply templates/node-cpu-hog-template.yml:9]: Yes, @ksatchit Sounds good.

**Review by chandankumar4 — APPROVED** [review]

File: templates/container-kill-template.yml [hunk 5]
```diff
@@ -0,0 +1,13 @@
+---
+variables:
+  APP_NS: default
+  APP_KIND: deployment 
+  APP_LABEL: %APPLICATION_LABEL%
+  TARGET_CONTAINER: %TARGET_CONTAINER_NAME%
+  EXPERIMENT_IMAGE: litmuschaos/ansible-runner:latest
```
Comment (ksatchit) [inline templates/container-kill-template.yml:7]: pin to versions

**Review by ksatchit — APPROVED** [review]

**Pull request merged**